static/index-no-inline.html

```diff
@@ -5,12 +5,16 @@
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0, shrink-to-fit=no">
   <title><%= htmlWebpackPlugin.options.title %></title>
 </head>
-<body class="xh-app">
+<body class="xh-app" style="margin: 0">
 
   <noscript>
       This application requires that JavaScript be enabled for this site in your browser.
   </noscript>
 
+  <div id="xh-preload-spinner" style="height: 100vh; width: 100vw; display: flex; align-items: center; justify-content: center;">
+      <object data="../public/spinner.png" type="image/png" style="width: 50px; height: 50px;"></object>
```

Comment:
Note this use of the `object` tag avoids a broken image icon display if the app has an older version of hoist-dev-utils.  (The div will still be there to hide in XH, regardless)